Makefile

```diff
@@ -188,7 +188,8 @@ INCLUDES_CF2 += -I$(STLIB)/STM32_USB_OTG_Driver/inc
 INCLUDES_CF2 += -Ideck/interface
 
 ifeq ($(USE_FPU), 1)
-PROCESSOR = -mcpu=cortex-m4 -mthumb -mfloat-abi=hard -mfpu=fpv4-sp-d16
+	PROCESSOR = -mcpu=cortex-m4 -mthumb -mfloat-abi=hard -mfpu=fpv4-sp-d16
+	CFLAGS += -fno-math-errno
```

Comment:
FYI, this must go in `CFLAGS` and not `PROCESSOR` because you pass the contents of `PROCESSOR` to `as`, which interprets `-f` flags rather differently.